jest.setup.js

```diff
@@ -1 +1,3 @@
-global.fetch = require("whatwg-fetch").fetch;
+global.fetch = require("jest-fetch-mock");
```

Comment:
hum why ?

Reply:
im getting some weird fetch error in circle ci `Jest did not exit one second after the test run has completed`, was trying this to see if it works, but doesnt seem so too :(